src/debug.ts

```diff
@@ -182,6 +252,33 @@ export async function debugTests(
   await deferredPromise.promise
 }
 
+function getTestProjectsInFolder(path: string, api: VitestFolderAPI, tree: TestTree, exclude: Set<string>) {
+  const folder = tree.getOrCreateFolderTestItem(api, path)
+  const items = tree.getFolderFiles(folder)
+  return items.map(item => (getTestData(item) as TestFile)).filter(testfile => !exclude.has(testfile.id)).map(tf => tf.project)
+}
+
+function getProjectsFromTests(fsPath: string, parentItem: vscode.TestItem | undefined, api: VitestFolderAPI, tree: TestTree, excluded: Set<string>): string[] {
+  const items = getTestProjectsInFolder(fsPath, api, tree, excluded)
+  if (items.length > 0) {
+    return items
+  }
+  // Climb up tree until entry with project is found
+  const parentPath = parentItem?.uri?.fsPath
+  if (parentPath) {
+    return getProjectsFromTests(parentPath, parentItem?.parent, api, tree, excluded)
+  }
+  return []
+}
+
+function getBrowserModeLaunchArgs(isPlaywright: boolean, config: { debuggerPort?: number; cliArguments?: string }): string {
+  const browser = !config.cliArguments?.includes('--browser') ? `--browser=${isPlaywright ? 'chromium' : 'chrome'}` : ''
+  // Only playwright provider supports --inspect currently
+  const inspect = isPlaywright && !config.cliArguments?.includes('--inspect') ? `--inspect=localhost:${config.debuggerPort ?? '9229'}` : ''
```

Comment:
Previously I was using `--inspect-brk`, but it seems that behavior is now consistent with `--inspect`. Now starting with `--inspect-brk` will pause on every file and not just the first, which is a bad experience.

I'm not really sure why this has changed; I reviewed release notes and recent PRs in Vite and Vitest but nothing jumped out.